test/performance/long-running-tests/README.md

```diff
@@ -0,0 +1,189 @@
+   ```
+    
+   The generated workload is a pod that sleeps for 20 seconds([see here](workloads/long-duration-exp-workload.yaml)). Proceed to the next step below and execute it in parallel.
+
+
+5. Open a new terminal and run the metrics collection script to collect the creation and status_update timestamps for the benchmark workload objects:
+
+   a) cd into the `test/performance/common` directory from your local copy of the KubeStellar repo, for example:
+   
+   ```bash 
+   cd $HOME/kubestellar/test/performance/common
+   ```
+
+   b) Run the metrics collection script:
+
+   ```bash
+   python3 metrics_collector.py <kubeconfig> <wds-context-name> <its-context-name> <wec-context-name> <number-of-namespaces> <output-directory> <exp-type> <number-of-pods> <watch-interval>
+   ```
+
+   For example:
+   
+   ```bash 
+   python3 metrics_collector.py $HOME/.kube/config wds1 its1 cluster1 1 $HOME/data l 10 660
+   ```
+
+   Observations: 
+      - The execution of the above python script requires the installation of the dependencies listed in `requirements.txt`.
+      - The script creates a custom controller that deletes a pod after reaching the completed state.
+      - The workload generation tool (i.e., clusterloader2) launched at step 4 will create another pod after `stepDelay` (default value: 1 minute) with a different name. 
+      - The above procedure guarantees that the running system is not overloaded. 
+
+   Below is a detailed explanation of the input parameters:
+   - `kubeconfig`: path to the kubeconfig file, e.g., `$HOME/.kube/config`
+   - `wds-context-name`: name of the context for the target WDS, e.g., `wds1`
+   - `its-context-name`: name of the context for the target ITS, e.g., `its1`
+   - `wec-context-name`: name of the context for the target WEC, e.g., `cluster1`
+   - `number-of-namespaces`: it must be set to `1` always. There is only one namespace created: `perf-test`. 
+   - `output-directory`: path to the directory for the output data files, e.g., `$HOME/data`
+   - `exp-type`: experiment type (e.g., `l`)
+   - `number-of-pods`: total number of pods to be created in the experiment. The same value configured at step 3-b. 
+   - `watch-interval`: time duration in seconds of your experiment (e.g., `660`). The same value configured at step 3-b.
+
+   The creation and status_update timestamps will be collected for the workload objects created in `wds1`, manifestwork & workstatus objects created in `its1`, and workload & appliedmanifestworks objects created in `wec1`, for example:
+
+   ```bash  
+   cd $HOME/data
+   tree
+   ```
+
+   Output:
+   ```console  
+      .
+      ├── 10-appliedmanifestwork.csv
+      ├── 10-manifestworks.csv
+      ├── 10-pod-wds.csv
+      ├── 10-pod-wec.csv
+      └── 10-workstatus.csv
+   ```
+
+   The generated output files have the following structure:
```

Comment:
You might want to mention that the separator character is tab.

Reply:
ok, update that sentence as following: 

```
 The generated output from the script are tab-delimited files with the following structure:
```